orttraining/orttraining/test/python/orttraining_test_orttrainer_bert_toy_onnx.py

```diff
@@ -19,6 +20,12 @@
 
 import _test_commons, _test_helpers
 
+ADAM_OPTIM_NAME = optim.AdamConfig().name
+LAMB_OPTIM_NAME = optim.LambConfig().name
+MOMENT_KEYS = ["Moment_1", "Moment_2"]
+UC_KEY = "Update_Count"
+STEP_KEY = "Step"
+SHARED_STATE_KEY = "shared_optimizer_state"
```

Comment:
I see this facilitating code-reuse, but still better not to use global vars. They contain implementation details that ideally users should never see or else they can have funny ideas and start using it too. Each method must be self contained and easily replaced

Although you can move this to `get_optim_state_from_state_dict`/`generate_dummy_optim_state`, consider using more idiomatic constructions when possible, such a `isinstance(optimizer, optim.AdamConfig)`, etc

Reply:
moved away from global vars for these tests.